EIPS/eip-1690.md

```diff
@@ -0,0 +1,174 @@
+---
+eip: 1690
+title: Contract Mortability Standard
+author: Felix Franz (@Felix-Franz)
+discussions-to: https://github.com/ethereum/EIPs/issues/1690
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-01-11
+requires: 173
+---
+
+## Simple Summary
+
+A standard that allows the owner of a contract to completely remove it.
+
+## Abstract
+
+For removing a smart contract the solidity function `selfdestruct(address);` needs to be called from inside the Contract.
+This function allowes to provide a address where the ether currently owned by the smart contract will be transferred before deleting the contract.
+It should be called from outside the contract by the owner only using a public function.
+
+## Motivation
+
+To completely remove a smart contract that is e.g. not needed anymore or was created by mistake, the functionality needs to be implemented in the contract.
+This EIP should standardize this functionality.
```

Comment:
Motivation should explain why a standard is necessary here.  It is unclear to me why this needs to be standardized as each contract is likely to have its own management tooling.  I find it highly unlikely that someone is going to author general purpose contract deletion tooling as that is a pretty application specific action.